packages/@react-facet/deferred-mount/src/index.spec.tsx

```diff
@@ -37,61 +70,63 @@ describe('DeferredMountWithCallback', () => {
     expect(container.firstChild).toContainHTML('<div>Should be rendered</div>')
   })
 
-  it('waits until previous deferred callback finishes', async () => {
-    jest.useFakeTimers()
-
-    const frames: (() => void)[] = []
-    const requestSpy = jest.spyOn(window, 'requestAnimationFrame').mockImplementation((frameRequest) => {
-      const id = idSeed++
-      const cb = () => {
-        frameRequest(id)
-      }
-      frames.push(cb)
-      return id
-    })
-
-    const runRaf = () => {
-      const cb = frames.pop()
-      if (cb != null) act(() => cb())
+  it('defers again after initial defer has completed', () => {
+    const DeferConditionally: React.FC<{ mountDeferred: boolean }> = ({ mountDeferred }) => {
+      return (
+        <>
```

Comment:
I would maybe add a print out of the `useIsDeffering()` hook value so you can verify that it's being set correctly, and then later run some `expects` on it. wdyt?
```suggestion
        <>
                <p>isDeferring: ${ useIsDeferring() } 
```

Reply:
Makes sense to me! I fixed it in my latest commit. Tried to follow the convention already in place in other tests for checking this.